modules/policy-identity-access-management.adoc

```diff
@@ -13,9 +13,7 @@ For a list of the available subprocessors, see the link:https://access.redhat.co
 
 [id="sre-access-all_{context}"]
 == SRE access to all {product-title} clusters
-SREs access {product-title} clusters through the web console or command-line tools. Authentication requires multi-factor authentication (MFA) with industry-standard requirements for password complexity and account lockouts. SREs must authenticate as individuals to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
-
-SREs access private clusters using an encrypted tunnel through a hardened SRE Support Pod running in the cluster. Connections to the SRE Support Pod are permitted only from a secured Red Hat network using an IP allow-list. In addition to the cluster authentication controls described above, authentication to the SRE Support Pod is controlled using SSH keys. SSH key authorization is limited to SRE staff and automatically synchronized with Red Hat corporate directory data. Corporate directory data is secured and controlled by HR systems, including management review, approval, and audits.
+SREs access {product-title} clusters through a proxy running in front of OSD clusters, The proxy will mint a Service Account in OSD cluster for SREs when they try to login to the cluster. SREs must authenticate as individuals to ensure auditability. All authentication attempts are logged to a Security Information and Event Management (SIEM) system.
```

Comment:
@wanghaoran1988 Is it worth clarifying to the customer that we don't use the web console or usual command-line login? I have seen some queries around this in the past from customers as secondary

Reply:
@chopraapooja Updated